nyu_event/settings.py

```diff
@@ -28,7 +28,7 @@
 DEBUG = True
 
 # ALLOWED_HOSTS = ['*','nyu-event-dev.us-east-1.elasticbeanstalk.com']
-ALLOWED_HOSTS = ["127.0.0.1", "postgres-dev.us-east-1.elasticbeanstalk.com"]
+ALLOWED_HOSTS = ["127.0.0.1", "main2.us-east-1.elasticbeanstalk.com"]

```

Comment:
[IMHO] we can keep both "postgres-dev.us-east-1.elasticbeanstalk.com" and "main2.us-east-1.elasticbeanstalk.com"

Reply:
ok